fvm/src/call_manager/default.rs

```diff
@@ -638,15 +634,19 @@ where
             },
         };
 
-        self.send_resolved::<K>(from, to, method, params, value, read_only)
+        self.actor_call_stack.push((to, entrypoint.func_name()));
+        let res = self.send_resolved::<K>(from, to, entrypoint, params, value, read_only);
+        self.actor_call_stack.pop();
+
+        res
     }
 
     /// Send with resolved addresses.
     fn send_resolved<K>(
```

Comment:
now that we're mostly down to nits, I'd rename all `send_*` methods on the call manager to `call_actor_*` to avoid confusion.